cms/static/js/views/active_video_upload_list.js

```diff
@@ -10,6 +11,7 @@ define(
                 "dragleave .file-drop-area": "dragleave",
                 "drop .file-drop-area": "dragleave"
             },
+            pageUnloadMessage: pageUnloadMessage,
```

Comment:
I did that solely so I could reference it in the tests..  Is that a no-no?
https://github.com/pomegranited/edx-platform/blob/master/cms/static/js/spec/views/active_video_upload_list_spec.js